spec/volumes_v1alpha2_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+ require_relative './shared_context'
+ 
+  require 'fog/kubevirt'
+ 
+  describe Fog::Compute do
+   before :all do
+     vcr = KubevirtVCR.new(
+       vcr_directory: 'spec/fixtures/kubevirt/volume/v1alpha2',
+       service_class: Fog::Compute
+     )
+     @service = vcr.service
+   end
+ 
+    it 'CRUD services' do
+     VCR.use_cassette('volumes_crud') do
+       begin
+         name = 'my-local-storage'
+         access_modes = [ 'ReadWriteOnce' ]
+         capacity = '1Gi'
+         labels = { type: 'local' }
+         storage_class = 'manual'
+         type = 'hostPath'
+         config = { path: "/mnt/data/datax", type: "Directory" }
+
+         @service.volumes.create(name: name,
+                                 access_modes: access_modes,
+                                 capacity: capacity,
+                                 labels: labels,
+                                 storage_class: storage_class,
+                                 type: type,
+                                 config: config)
+ 
+          volume = @service.volumes.get(name)
+          expect(volume.name).to eq('my-local-storage')
```

Comment:
Travis fails here:

> NoMethodError: undefined method `eq' for #<#<Class:0x0000000002cb4650>:0x0000000002cde2e8>
>    /home/travis/build/fog/fog-kubevirt/spec/volumes_v1alpha2_spec.rb:35:in `block (3 levels) in <top (required)>'

Reply:
I executed the tests locally with rspec while 'bundle exec rake' uses minitest. I've updated the test accordingly.